src/bmpimage.cpp

```diff
@@ -31,25 +31,25 @@ EXIV2_RCSID("@(#) $Id$")
 // included header files
 #include "config.h"
 
-#include "bmpimage.hpp"
-#include "image.hpp"
 #include "basicio.hpp"
+#include "bmpimage.hpp"
 #include "error.hpp"
 #include "futils.hpp"
+#include "image.hpp"
 
 // + standard includes
-#include <string>
 #include <cstring>
 #include <iostream>
+#include <string>
 
 // *****************************************************************************
 // class member definitions
-namespace Exiv2 {
-
+namespace Exiv2
+{
     BmpImage::BmpImage(BasicIo::AutoPtr io)
-        : Image(ImageType::bmp, mdNone, io)
+          : Image(ImageType::bmp, mdNone, io)
     {
-    } // BmpImage::BmpImage
+    }  // BmpImage::BmpImage
```

Comment:
Why two blanks here?

Reply:
Good point. We can control this with the field:
```
SpacesBeforeTrailingComments: 2
```
I will change it to 1.

But ... should not we just remove these comments ? I think they are useless. If there are functions so long that somebody can think it is a good idea to have such comments, I would rather think that we should refactor the code to have smaller functions that can be read easily.

Reply:
Indeed, good point to get rid of useless comments.

Reply:
Yes, a good editor will tell you anyway and if you actually need such comments the function is too long. However, I'd keep these comments for namespaces and `#ifdefs`